bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -156,10 +156,10 @@ object BagsState {
       (RunningEditedState(k), ioUnit)
 
     // Successful AGS lookup while running (and not edited).  Apply the update
-    // and move to IdleState.
+    // and move to IdleState. If the PA changes, update the hash.
     private[ags] override def succeed(results: Option[AgsStrategy.Selection]): StateTransition = {
       val h = results.fold(hash) { r =>
-        if (r.posAngle === ctx.getPositionAngle) hash
+        if ((r.posAngle.toDegrees - ctx.getPositionAngle.toDegrees).abs < 1e-4) hash
```

Comment:
We can probably require it to be even closer.  This is the kind of problem we might run into:

````
scala> val x = 35.77
x: Double = 35.77

scala> x + 180.0 - 180.0
res0: Double = 35.77000000000001
````

So even `1e-10` seems fine?